map/index.php

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html lang="de-DE">
+  <head>
+
+      <?php require_once("./header.php") ?>
+      
+      <title>RailwayStations</title>
+
+      <link rel="stylesheet" href="css/leaflet.css" />
+      <link rel="stylesheet" href="css/MarkerCluster.css" />
+      <link rel="stylesheet" href="css/MarkerCluster.Default.css">
+
+  </head>
+<body>
+
+<?php
+require_once("./navbar.php");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["require_once" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/app/RailwayStations/RSWebsite/pullRequest?prid=4676723)